src/content/docs/en/hardware/surface.mdx

```diff
@@ -5,39 +5,49 @@ description: "Information about Ultramarine Linux on Microsoft Surfaces"
+| Surface Pro 3                | Yes                     | Boots without, but things may be broken                                                                                                            | ✓             |
+| Surface Pro 4                | Yes                     |                                                                                                                                                    |               |
+| Surface Pro (5th Generation) | Yes                     | Nearly identical to Pro 4                                                                                                                          | ✓             |
+| Surface Pro 6                | Yes                     | Nearly identical to 5th gen                                                                                                                        |               |
+| Surface Pro 7                | Yes                     |                                                                                                                                                    |               |
+| Surface Pro 7+               | Yes                     |                                                                                                                                                    |               |
+| Surface Pro 8                | Yes                     |                                                                                                                                                    |               |
+| Surface Pro 9                | Yes                     |                                                                                                                                                    |               |
+| Surface 3                    | Yes                     | Battery and touch will not work without                                                                                                            |               |
+| Surface Go                   | No                      | Cameras and LTE will not work without Surface Kernel                                                                                               | ✓             |
+| Surface Go 2                 | No                      | Same as Surface Go                                                                                                                                 | ✓             |
+| Surface Go 3                 | No                      |                                                                                                                                                    |               |
+| Surface Go 4                 | Yes                     | UFS Storage, requires some manual intervention to get working                                                                                      |               |
```

Comment:
Manual intervention guide should be provided below this chart, along with any other manual workarounds needed?

Reply:
I would talk about what surface_aggregator is/how to use it

Reply:
> Manual intervention guide should be provided below this chart, along with any other manual workarounds needed?

they should go to surface wiki imo, but maybe

> I would talk about what surface_aggregator is/how to use it
I can make a mention